pest/src/position.rs

```diff
@@ -183,23 +214,23 @@ impl<'i> Position<'i> {
                 None => 0
             }
         };
-
-        let end = if self.input.is_empty() {
+        let input_bytes = self.input.as_bytes();
```

Comment:
Maybe call this `bytes` for consistency?

Reply:
Fixed this [here](https://github.com/pest-parser/pest/pull/303/commits/5024c4029391e2eb554f16bb704d4e6fe3e507ff#diff-082ec1879c86e477c78290d490f0e6f6R217